data/projects/bear_updates.md

```diff
@@ -1,26 +1,42 @@
 This project is about improving the general quality of coala's linter bears.
 
-Linter bears in coala are oftentimes not receiving enough love.
-They work but not necessarily on all platforms. They are documented, but not
-necessarily understandable. They are tested but lack corner cases or
+Since it is very easy to write them even for very inexperienced contributors, linter bears in coala often suffer from mediocre quality,
+especially once they made it into the codebase.
+They do work, but not necessarily on all platforms or for all usecases. They are documented, but not
+necessarily understandable. They are tested, but lack corner cases or
 exploratory tests. [See existing issues](https://github.com/coala/coala-bears/issues?q=is%3Aopen+is%3Aissue+label%3Aarea%2Flintbears)
 
-Task of this project is to solve those issues for as many bears as possible and
-derive strategies to avoid those issues in the future.
+Students working on this project familiarize themselves in-depth with the working of the `linter`
+decorator and the existing linter bears. They identify systematic issues with the way linter bears are
+developed and maintained and work to fix those issues and improve and amend [existing documentation](http://api.coala.io/en/latest/Developers/Writing_Linter_Bears.html)
+to avoid them in the future.
```

Comment:
This could probably have more punch to it. What would be the great outcome of this project? Probably something like a wider range of nice working bears and thus nobody has to encounter non working bears when using random bears we don't usually use